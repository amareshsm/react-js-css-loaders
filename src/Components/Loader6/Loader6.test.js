import React from 'react'
import renderer from 'react-test-renderer'
import { Loader6 } from '.'
it('renders correctly', () => {
  const tree = renderer.create(<Loader6 />).toJSON()
  expect(tree).toMatchSnapshot()
})
