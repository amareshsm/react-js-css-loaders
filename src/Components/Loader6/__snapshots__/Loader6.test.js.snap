// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style>
    [object Object]
  </style>,
  <div
    className="loader-div six"
  >
    <div
      className="loader6"
    >
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>,
]
`;
